server/s3_put_object_action.cc

```diff
@@ -58,7 +58,20 @@ S3PutObjectAction::S3PutObjectAction(
   old_object_oid = {0ULL, 0ULL};
   old_layout_id = -1;
   new_object_oid = {0ULL, 0ULL};
-
+  fault_mode_active = false;
```

Comment:
why not initialize them in the initializer list !!

Reply:
yes, will do in initialize list.